treeherder/model/error_summary.py

```diff
@@ -82,13 +92,16 @@ def bug_suggestions_line(err, term_cache=None):
 
 
 def get_mozharness_substring(line):
+    """Strip possible Moz harness bits from the given line."""
```

Comment:
`mozharness` is all one word (https://dxr.mozilla.org/mozilla-central/source/testing/mozharness)

Reply:
oops, bad habit! Fixed and rebased in.